controllers/registries/replication_controller.go

```diff
@@ -98,12 +98,9 @@ func (r *ReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
 
 	// Check the Harbor API if it's reporting as healthy
-	instanceIsHealthy, err := internal.HarborInstanceIsHealthy(ctx, harborClient)
+	err = internal.AssertHealthyHarborInstance(ctx, harborClient)
 	if err != nil {
-		return ctrl.Result{}, err
-	}
-	if !instanceIsHealthy {
-		return ctrl.Result{RequeueAfter: 30 * time.Second}, nil
+		return ctrl.Result{RequeueAfter: 10 * time.Second}, err
```

Comment:
It was already like that before, but RequeueAfter is obsolete here, since an err! = nil is requeued anyway 
```suggestion
		return ctrl.Result{}, err
```